docker-compose.yml

```diff
@@ -39,9 +42,64 @@ services:
         extra_hosts:
             # just if you have a local CKAN instance running in _nginx_ host in you local machine
             # export HOST_IP=`ip -4 addr show scope global dev docker0 | grep inet | awk '{print \$2}' | cut -d / -f 1`
-            - "nginx:${HOST_IP}"
+            - "ckan:${HOST_IP}"
         healthcheck:
```

Comment:
no need to add healthchecks in a dev docker-compose.

Reply:
[removed](https://github.com/datopian/ckan-ng-harvest/commit/d65a2624edea72a7c26b365535c27922d73c7b40)